simson/common/data_extrapolations.py

```diff
@@ -20,6 +20,23 @@ def predict(self):
         pass
 
 
+class LinearExtrapolation(Extrapolation):
+
+
+
```

Comment:
Unnecessary white spaces, but otherwise this looks great as an Extrapolation child :)